MPI/microsoft-mpi.md

```diff
@@ -40,8 +40,8 @@ Earlier versions of MS-MPI are available from the [Microsoft Download Center](ht
 ## Microsoft High Performance Computing Resources
 
   - Featured tutorial: [How to compile and run a simple MS-MPI program](https://blogs.technet.com/b/windowshpc/archive/2015/02/02/how-to-compile-and-run-a-simple-ms-mpi-program.aspx)
-  - Featured guide: [Set up a Windows RDMA cluster with HPC Pack and A8 and A9 instances to run MPI applications](https://azure.microsoft.com/documentation/articles/virtual-machines-windows-hpcpack-cluster-rdma/)
-  - [Microsoft High Performance Computing for Developers](https://msdn.microsoft.com/en-us/library/ff976568.aspx)
+  - Featured guide: [Set up a Windows RDMA cluster with HPC Pack and A8 and A9 instances to run MPI applications](https://github.com/uglide/azure-content/blob/master/articles/virtual-machines/virtual-machines-windows-classic-hpcpack-rdma-cluster.md)
```

Comment:
This doc is old and not even formally archived, so let's just remove the entire line. Thanks!